src/components/Link/Link.scss

```diff
@@ -13,4 +13,24 @@
         color: var(--link-color-hover);
         text-decoration: var(--link-decoration-hover);
     }
+
+    &--context {
+        @each $context in (brand, neutral, primary, accent, info, good, warning, bad) {
```

Comment:
Would it not make sense to define this repeating array as a variable, kind of like what `mixins/colors.scss` does? The react code already does that. IMHO it's better to have context behave consistently between different components, instead of having to think "which context do we _really_ need now". And to find out later that some existing component implements all colors _except_ the one you need...